src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -1451,6 +1484,11 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
                 () => (this.newlyAdded = false),
                 STUDY_VIEW_CONFIG.thresholds.chartHighlight
             );
+        } else if (this.chartChanged) {
```

Comment:
currently when changing a chart type, the highlight doesn't go away. look into the mechanism behind this and figure out why